app/src/main/res/values/strings_intro.xml

```diff
@@ -3,31 +3,32 @@
     <string name="intro_slide1_title">Let\'s get started</string>
     <string name="intro_slide1_desc">Welcome to <b>andOTP</b>, this wizard will guide you through
         the initial setup. Please pay attention and read carefully otherwise you could lose data!
-        \n\nDon\'t worry, you can still change any of the options later in the <b>Settings</b>.</string>
+        \n\nDon\'t worry, you can still change any of the options later in the <b>Settings</b> page.</string>
 
-    <string name="intro_slide2_desc">To ensure the security of your accounts <b>andOTP</b> only
+    <string name="intro_slide2_desc">To ensure the security of your accounts, <b>andOTP</b> only
         stores them in encrypted data files. Here you can choose which method of encryption will be
         used.</string>
     <string name="intro_slide2_desc_keystore">The KeyStore is a system component of Android for
-        securely storing cryptographic keys. The advantage of this method is that the keys are
-        stored separated from the data files and can be backed by hardware cryptography (if your
-        device supports it). However, as the keys are not stored with the apps data, <b>this method
-        prevents external backup solutions (like Titanium) from working</b>. If you choose this
-        method you will have to rely on the internal backup functions provided by <b>andOTP</b>.
+        securely storing cryptographic keys. The advantage of using this method is that the keys are
+        stored separated from the data files of andOTP and can be backed up by hardware cryptography (if your
+        device supports it). However, as the keys are not stored with the andOTP\'s data files, <b>this method
```

Comment:
We should probably remove the `the` if we use `andOTP's`.

Reply:
Ok. Point noted. Would effect the chances ASAP